src/routes/view_api_monitor_detail/index.js

```diff
@@ -0,0 +1,126 @@
+import { h } from 'preact';
+import { 
+  Box,
+  Button,
+  ButtonGroup,
+  Flex,
+  Heading,
+  Select,
+  Spacer,
+  Spinner } from '@chakra-ui/react'
+import { useEffect, useState } from 'preact/hooks'
+import { route } from 'preact-router';
+import style from './style.css';
+import AlertComponent from '../../components/alertComponent';
+import axios from 'axios';
+import BASE_URL from '../../config/api/constant.js';
+import ROUTE from '../../config/api/route.js';
+import SuccessRatePercentageChart from '../../components/success_rate_percentage_chart';
+import ResponseTimeChart from '../../components/response_time_chart';
+import { getUserToken } from '../../config/api/auth';
+
+const ViewAPIMonitorDetail = ({id}) => {
+  const options = [
+    { value: "30MIN", label: "30 Minutes Ago" },
+    { value: "60MIN", label: "1 Hour Ago" },
+    { value: "180MIN", label: "3 Hours Ago" },
+    { value: "360MIN", label: "6 Hours Ago" },
+    { value: "720MIN", label: "12 Hours Ago" },
+    { value: "1440MIN", label: "24 Hours Ago" },
+  ];
+  
+  const [detail, setDetail]=useState([])
+  const [deletePopup, setDeletePopup] = useState(false) 
+  const [isLoadingDelete, setIsLoadingDelete] = useState(false)
+  const [selectValue, setSelectValue] = useState("30MIN")
+
+  const onDelete = () => {
+    setIsLoadingDelete(true)
+    axios.delete(`${BASE_URL}/monitor/${id}`, {
+      headers: {
+        Authorization: `Token ${getUserToken()}`
+      } 
+    }).then(() => {
+      route(ROUTE.DASHBOARD)
+      setIsLoadingDelete(false)
+    })
+  }
+
+  useEffect(() => {
+    axios.get(`${BASE_URL}/monitor/${id}/detail`, {
```

Comment:
```suggestion
    axios.get(`${BASE_URL}/monitor/${id}/`, {
```

ini kaykanya url nya bakal kayak gini aja fi